azkaban-common/src/main/java/azkaban/utils/DependencyTransferException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.utils;
+
+/**
+ * Indicates that a dependency failed to successfully upload or download.
+ */
+public class DependencyTransferException extends RuntimeException {
```

Comment:
If you are adding this exception and others like it, better make the best use of them by adding some additional info specific to this exception.